documentation/src/main/asciidoc/topics/ref_server_realm_ldap.adoc

```diff
@@ -11,12 +11,27 @@ LDAP servers can have different entry layouts, depending on the type of server a
 It is beyond the scope of this document to provide examples for all possible configurations.
 ====
 
-.Endpoint authentication mechanisms
+[IMPORTANT]
+====
+The principal for LDAP connections must have necessary privileges to perform
+LDAP queries and access specific attributes.
+====
 
-When you configure {brandname} Server to use an LDAP realm, you can configure endpoints to use the following authentication mechanisms:
+As an alternative to verifying user credentials with the `direct-verification` attribute, you can specify an LDAP password with the `user-password-mapper` element.
```

Comment:
"you can specify the which LDAP attribute to use when validating the password with the `user-password-mapper` element.